axelrod/strategy_transformers.py

```diff
@@ -141,6 +153,27 @@ def __repr__(self):
                     prefix = ', '
                 return name
 
+            def reduce_for_decorated_class(self_):
+                class_module = import_module(self_.__module__)
+                import_name = self_.__class__.__name__
+
+                if player_can_be_pickled(self_):
```

Comment:
i'm not sure how.  no error is raised in the __reduce__ function, so it wouldn't be able to catch it, right?  since this is called by the pickler, that is where you'd need the try/catch.